.github/workflows/platform.linter.yml

```diff
@@ -1,6 +1,10 @@
 name: '.Platform: Linter'
 
-on: [pull_request]
+on:
```

Comment:
You sure? Then we won't get the feedback on the PRs anymore

Reply:
PRs to main, same as Broken links pipeline.